### PR TITLE
Add more DB stats

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds more DB statistics (Comments, Annotations, etc.)

### CODE REVIEW

File: extensions/skyportal/skyportal/handlers/api/db_stats.py [hunk 1]
```diff
@@ -82,116 +91,168 @@ def get(self):
...
+                )
+            )[0][0]
 
-        for survey in ("PGIR", "ZTF"):
-            utc_now = datetime.datetime.utcnow()
-            jd_start = Time(
-                datetime.datetime(utc_now.year, utc_now.month, utc_now.day)
-            ).jd
-            query_alerts_count = {
+        with self.Session() as session:
```
Comment (guynir42) [inline extensions/skyportal/skyportal/handlers/api/db_stats.py:106]: Why not continue with the same session as above?

File: extensions/skyportal/skyportal/handlers/api/db_stats.py [hunk 2]
```diff
@@ -82,116 +91,168 @@ def get(self):
...
+                sa.select(CronJobRun.script).distinct()
+            ).all()
+            for script in cron_job_scripts:
+                cron_job_run = session.execute(
+                    sa.select(CronJobRun)
+                    .where(CronJobRun.script == script)
+                    .order_by(CronJobRun.created_at.desc())
+                ).first()
+                if cron_job_run is not None:
+                    (cron_job_run,) = cron_job_run
```
Comment (guynir42) [inline extensions/skyportal/skyportal/handlers/api/db_stats.py:183]:
```suggestion
                cron_job_run = session.scalars(
                    sa.select(CronJobRun)
                    .where(CronJobRun.script == script)
                    .order_by(CronJobRun.created_at.desc())
                ).first()
```

File: extensions/skyportal/skyportal/handlers/api/db_stats.py [hunk 3]
```diff
@@ -82,116 +91,168 @@ def get(self):
...
+            cand = session.scalars(
+                sa.select(Candidate)
+                .where(Candidate.obj_id.notin_(sa.select(Source.obj_id).subquery()))
+                .order_by(Candidate.created_at)
+            ).first()
+            data["Oldest unsaved candidate creation datetime"] = (
+                cand.created_at if cand is not None else None
+            )
+            data["Latest cron job run times & statuses"] = []
+            cron_job_scripts = session.execute(
```
Comment (guynir42) [inline extensions/skyportal/skyportal/handlers/api/db_stats.py:173]:
```suggestion
            cron_job_scripts = session.scalars(
```

File: extensions/skyportal/skyportal/handlers/api/db_stats.py (hunk 1 above)
Comment (mcoughlin) [reply extensions/skyportal/skyportal/handlers/api/db_stats.py:106]: thats a special DBSession() that executes sql directly (same trick we use in Skyportal).

Comment (guynir42) [reply extensions/skyportal/skyportal/handlers/api/db_stats.py:106]: right, didn't notice it was a different kind of session.

**Pull request merged**